normandy/recipes/management/commands/update_addon_urls.py

```diff
@@ -33,17 +39,23 @@ def handle(self, *args, new_hostname, **options):
                 )
                 continue
 
-            parsed_url = urlparse(arguments["addonUrl"])
-            if parsed_url.netloc == new_hostname:
+            filename = get_filename_from_url(arguments["addonUrl"])
+
+            if filename not in extension_by_filename:
+                self.stderr.write(
+                    f"Warning: Recipe {rev.recipe.id} revision {rev.id} has an addonUrl that does "
+                    f"not match any in the database.",
+                    ending="\n",
```

Comment:
I know it was like that before (line 38) but why this extra `ending="\n"`. It's added by default for you. 

> "Note also that you don’t need to end messages with a newline character, it will be added automatically"

https://docs.djangoproject.com/en/2.1/howto/custom-management-commands/

`self.stderr.write("foo")` is almost the same as `sys.stderr("foo\n").

Reply:
I also don't know why this was here, and decided not to think about it, in case there was something complicated. Now that you've thought about it and done the research, I'll remove it.